README.md

```diff
@@ -14,7 +14,7 @@ betterlockscreen --lock blur
 
```

Comment:
General note: Do we want to include a contact/support note with details for irc?

Reply:
Sure